symphony-bdk-examples/bdk-app-spring-boot-example/build.gradle

```diff
@@ -10,5 +10,3 @@ dependencies {
     implementation project(':symphony-bdk-spring:symphony-bdk-app-spring-boot-starter')
     implementation 'org.springframework.boot:spring-boot-starter-security'
 }
-
-uploadArchives.enabled = false
```

Comment:
Why did you remove that ? Is it already disabled in the root build.gradle file ?

Reply:
This is coming from an old plugin that we don't use anymore (and now this API has been removed in Gradle 7)
Now it works with convention plugins where we explictly apply the id 'bdk.java-publish-conventions' plugin
So we don't have to disable the upload manually